utils/filter.h

```diff
@@ -143,8 +145,11 @@ void uftrace_setup_caller_filter(char *filter_str, struct symtabs *symtabs,
 				 struct rb_root *root,
 				 struct uftrace_filter_setting *setting);
 void uftrace_setup_hide_filter(char *filter_str, struct symtabs *symtabs,
-				 struct rb_root *root,
-				 struct uftrace_filter_setting *setting);
+			       struct rb_root *root,
+			       struct uftrace_filter_setting *setting);
+void uftrace_setup_loc_filter(char *filter_str, struct symtabs *symtabs,
+			      struct rb_root *root, enum filter_mode *mode,
```

Comment:
Sorry, I'll recheck the indentation of this block.

Reply:
Thanks!